domains/lousybot.vulpine.json

```diff
@@ -7,7 +7,7 @@
         },
     
         "record": {
-            "A": ["69.30.249.53"]
+            "A": ["69.197.135.205"]
```

Comment:
Unable to view, site, please provide a preview.

Reply:
It's a transcript system, here is a preview (it's written in Italian tho): http://dono-01.danbot.host:2251/transcript/transcript-partnership-31.html

Reply:
Btw for me it doesn't open by clicking on the link try copy-pasting it